package.json

```diff
@@ -15,9 +15,7 @@
     "test": "jest",
     "test:core": "npm test --prefix packages/core",
     "dev": "node ./scripts/dev.js",
-    "ci": "npm run build && npm run test",
     "changeset": "changeset",
-    "prerelease": "npm run build && npm run test",
```

Comment:
"pre*" scripts dont work with all package managers, so I've decided to just move this directly to a github action